### PR TITLE
Protobuf

### PR DESCRIPTION
**Pull request by ocefpaf**
See https://github.com/conda-forge/protobuf-feedstock/issues/75

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Can we hold off on this for a little bit? I need to fix the build bump stuff

**Comment by ocefpaf** [comment]
> Can we hold off on this for a little bit? I need to fix the build bump stuff

Sure. There is no rush. I can also build the older version with py38.

**Comment by ocefpaf** [comment]
> Can we hold off on this for a little bit? I need to fix the build bump stuff

@CJ-Wright I think this should be OK now, right?

**Comment by CJ-Wright** [comment]
The issue is this line: https://github.com/regro/cf-scripts/blob/master/conda_forge_tick/migrators.xsh#L1325 we set the build number bump to zero for the python migration. We need to set it back to 1 for this migration, or get this set from the migration yaml itself. Part of https://github.com/regro/cf-scripts/pull/689/files handled this.

File: recipe/migrations/protobuf310.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+migrator_ts: 1572797096
+__migrator:
+  kind:
+    version
+  # Only change the migration_number if the bot messes up,
+  # changing this will cause a complete rerun of the migration
+  migration_number:
+    1
+  build_number:
+    1
```
Comment (ocefpaf) [inline recipe/migrations/protobuf310.yaml]: @CJ-Wright is this correct?

Comment (CJ-Wright) [reply recipe/migrations/protobuf310.yaml]: yes

Comment (ocefpaf) [reply recipe/migrations/protobuf310.yaml]: So, merge :grimacing:

Comment (CJ-Wright) [reply recipe/migrations/protobuf310.yaml]: No, not yet need to fix bot first

**Comment by ocefpaf** [comment]
> This is pending a fix to the bot

No problem. I thought the issue was with this PR. I'll try to create an older build of `protobuf` instead to unblock some of the py38 migration PRs.

**Comment by CJ-Wright** [comment]
Yeah the issue here is that we can't configure the build number stuff. I'm working on fixing it.

**Comment by ocefpaf** [comment]
> Yeah the issue here is that we can't configure the build number stuff. I'm working on fixing it.

No rush. We actually may need the protobuf stuff for 3.8 anyways.

**Comment by CJ-Wright** [comment]
I want to get this in hand. It is very frustrating that we can't run these migrations side by side.

**Comment by ocefpaf** [comment]
> I want to get this in hand. It is very frustrating that we can't run these migrations side by side.

I know :smile:, just no pressure from my side. I own you way too many beers already.

**Pull request merged**